.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -56,8 +58,37 @@ stages:
             name: "ValidateState"
             displayName: "Validate State"
             retryCountOnTaskFailure: 3
-  - stage: connectivity_tests
+  - stage: restart_nodes
     dependsOn: validate_state
+    displayName: "Restart Node"
+    jobs:
+      - job: restart_nodes
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                echo "Scale up the pods and immediated restart the nodes"
+                make -C ./hack/swift set-kubeconf AZCLI=az CLUSTER=${RESOURCE_GROUP}
+                make -C ./hack/swift azcfg AZCLI=az REGION=$(LOCATION)
+                echo "Scaling the pods down to 100 per node"
+                bash ./hack/scripts/scale_deployment.sh -n 0 -u 1000 -s
+                echo "Restarting the nodes"
+                vmss_name=$(az vmss list -g MC_${RESOURCE_GROUP}_${RESOURCE_GROUP}_$(LOCATION) --query "[].name" -o tsv)
+                make -C ./hack/swift restart-vmss AZCLI=az CLUSTER=${RESOURCE_GROUP} REGION=$(LOCATION) VMSS_NAME=$vmss_name
+                bash ./hack/scripts/scale_deployment.sh -n 0 -u 1000 -c

```

Comment:
why are we calling scale script again?

Reply:
Similarly `-c` will wait until the deployment of 1000 pods is done. The same script is used.
I added the description of what the script can do. And checking the deployment status and deploying pods(scaling pods) can be done via passing flags to the script.
If we don't wait for the deployment to finish after the restart, the validate state script will fail as few of pods( sometimes) are still being created